assets/traefik/traefik.tpl

```diff
@@ -1,47 +1,72 @@
-graceTimeOut = "10s"
-debug = false
-checkNewVersion = false
-logLevel = "INFO"
-
-# If set to true invalid SSL certificates are accepted for backends.
-# Note: This disables detection of man-in-the-middle attacks so should only be used on secure backend networks.
-# Optional
-# Default: false
-#
-# InsecureSkipVerify = true
+[entryPoints]
+  [entryPoints.web]
+    address = ":80"
+    [entryPoints.web.http]
+      [entryPoints.web.http.redirections]
+        [entryPoints.web.http.redirections.entryPoint]
+          to = "websecure"
+          scheme = "https"
 
-defaultEntryPoints = ["http", "https"]
+  [entryPoints.websecure]
+    address = ":443"
 
-[entryPoints]
-  [entryPoints.http]
-  address = ":80"
-    [entryPoints.http.redirect]
-    entryPoint = "https"
-  [entryPoints.https]
-  address = ":443"
-    [entryPoints.https.tls]
-    MinVersion = "VersionTLS11"
-    CipherSuites = ["TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256", "TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384", "TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305"]
+[providers]
+  providersThrottleDuration = "2s"
+  [providers.docker]
+    watch = true
+    endpoint = "unix:///var/run/docker.sock"
+    exposedByDefault = true
+    swarmMode = true
+    swarmModeRefreshSeconds = "15s"
+    httpClientTimeout = "0s"
+    defaultRule = "Host(`{{ trimPrefix `/` .Name }}.${domain}`)"
+  [providers.file]
+    filename="/etc/traefik/vocab.toml"
+    watch=true
 
-[acme]
-email = "bgates@microsoft.com"   # FIXME
-storage = "/acme/acme.json"
-entryPoint = "https"
-acmeLogging = true
+[tls.options]
+  [tls.options.default]
+    minVersion = "VersionTLS10"
+    cipherSuites = [
+      "TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256",
+      "TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384",
+      "TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305",
+      "TLS_RSA_WITH_AES_128_CBC_SHA",
+      "TLS_RSA_WITH_AES_256_CBC_SHA"
+   ]
 
-[acme.dnsChallenge]
-provider = "godaddy"
-delayBeforeCheck = 0
+[api]
+  debug = false
+  dashboard = true
+  insecure = true
 
-[[acme.domains]]
-main = "*.${domain}"
+[log]
+  level = "DEBUG"
```

Comment:
Since we don't even aggregate those logs it's just gonna sit there and grow... Set it to INFO at most?